src/pages/Profile/Profile.jsx

```diff
@@ -1,19 +1,27 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-// import person from '../../images/person.png';
-// ^^ blank avatar for profile pic, uncomment the above line to use
+import person from '../../images/person.png';
 import './Profile.css';
+import ProfileStat from './ProfileStat';
 
 const Profile = ({ firstName, lastName, stats }) => {
   return (
     <div className="profile-page">
-      {/* TODO:
-      Complete the profile page here.
-      For the profile picture, use the avatar placeholder image that has
-      already been imported for you on line 3.
-      For the name & stats to display, use the props (firstName, lastName, and stats)
-      that have already been declared. */}
-      {firstName} {lastName} {stats}
+      <div className="main-profile">
+        <div className="avatar">
+          <img src={person}></img>
```

Comment:
[nit] if our closing tag immediately follows our opening tag we should use a self closing tag, e.g. `<img .... .... />` so that we don't need two tags to convey the same